File: apps/ios/dakka/Config/AppConfig.swift
import Foundation

enum AppConfig {
    /// Базовый URL API — переопределяется через `API_URL` в Info.plist при необходимости.
    static let apiBaseURL: URL = {
        if let raw = Bundle.main.object(forInfoDictionaryKey: "API_URL") as? String,
           let url = URL(string: raw) {
            return url
        }
        return URL(string: "https://akkdmsg.online/api")!
    }()

    /// Хост для socket.io (тот же origin без /api).
    static let socketBaseURL: URL = {
        if let raw = Bundle.main.object(forInfoDictionaryKey: "SOCKET_URL") as? String,
           let url = URL(string: raw) {
            return url
        }
        return URL(string: "https://akkdmsg.online")!
    }()
}

File: apps/android/app/src/main/java/online/akkdmsg/dakka/data/AppConfig.kt
package online.akkdmsg.dakka.data

import online.akkdmsg.dakka.BuildConfig

/**
 * Конфигурация приложения. URL'ы можно переопределить через BuildConfig flavors
 * или Gradle properties в будущем.
 */
object AppConfig {
    /** REST API base URL (без trailing slash). */
    const val API_BASE_URL: String = "https://akkdmsg.online/api"

    /** WebSocket base — без /socket.io/, socket.io-client сам добавит. */
    const val SOCKET_BASE_URL: String = "https://akkdmsg.online"

    /** Версия для User-Agent header. */
    val versionName: String = BuildConfig.VERSION_NAME
}
